modules/hcp-ec2-client/templates/hashicups.nomad

```diff
@@ -38,19 +49,23 @@ job "hashicups" {
   group "product-public-api" {
     network {
       mode = "bridge"
+      port "http" {
+        static = 8080
+      }
     }
 
     service {
       name = "product-public-api"
-      port = "8080"
+      port = NOMAD_PORT_http
+      tags = ["urlprefix-/"]
```

Comment:
What is `urlprefix` doing?

Reply:
urlprefix is used by fabio load balancer in nomad setups.  We can take it out.